### ISSUE TITLE
Expected integer for height but received -24 of type number - Error report for exile-diary 1.1.0

### ISSUE THREAD

**Issue by shakesx** [Qt-dev/exile-diary#182]
Error:
```
Error: Expected integer for height but received -24 of type number
    at Object.invalidParameterError (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\node_modules\sharp\lib\is.js:135:10)
    at Sharp.<anonymous> (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\node_modules\sharp\lib\resize.js:475:16)
    at Array.forEach (<anonymous>)
    at Sharp.extract (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\node_modules\sharp\lib\resize.js:470:38)
    at C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ScreenshotWatcher.js:285:30
    at step (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ScreenshotWatcher.js:33:23)
    at Object.next (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ScreenshotWatcher.js:14:53)
    at fulfilled (C:\Users\MJ\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ScreenshotWatcher.js:5:58)
```
OS: Windows NT 10.0.19045
App: exile-diary 1.1.0

================

Happening when I screenshot map mods

**Comment by Qt-dev** [comment]
Ha, I know this one. This happens when it fails to find the mods box (that dark background box with the mods in blue in it). It tends to happen to me when I play in a resolution that is a bit too wide and I have no bar on the sides, or when I try to capture mods of Invitations and all that. 

Could any of that have been happening?

**Comment by shakesx** [comment]
I have been running cemetery with the same mods, same settings, etc. Running in Windowed Fullscreen 1920x1080. Cemetery has a dark background and has been working previously so I don't think it's that. Will test some more maps tonight.

**Comment by Qt-dev** [comment]
Thanks. I have not touched all that in a while so that is probably some bad detection that's been broken for a bit.
If you can add a screenshot that failed (kill ED and then take a screenshot or something) I can try to see how I could tweak the algorithm to be better at reading it

**Comment by Qt-dev** [comment]
This should be fixed with 1.2.0 (and the next ones after that). Feel free to reopen if it fails again
